src/client/components/DashboardCalendar.tsx

```diff
@@ -58,62 +60,70 @@ class CalendarEvent {
 
 function SelectableCalendar ({ localizer }: Props) {
     const [events, setEvents] = useState([{allDay:true, title: "Today"}] as CalendarEvent[]);
+    const [modalOpen, setModalOpen] = useState(false)
+
 
     // prompting user for new event
     const handleSelect = ({ start, end }) => {
-        const title = window.prompt('New Event name')
-
-        // If the title of event exists then, create the event
-        if (title) {
-            let newEvent = {} as CalendarEvent;
-            newEvent.start = moment(start).toDate();
-
-            // TODO :: Fix 'All Day' Agenda for All day events
-            newEvent.end = moment(end).add(1, "seconds").toDate();
-            newEvent.title = title;
-
-            // Tracking previous events
-            setEvents([
-              ...events,
-              newEvent
-            ])
-        }
+
+        setModalOpen(true);
+    //     const title = window.prompt('New Event name')
+    //     // If the title of event exists then, create the event
+    //     if (title) {
+
+    //         // let newEvent: CalendarEvent = {
+    //         //     start: moment(start).toDate()
+    //         //     end: moment(end).add(1, "seconds").toDate()
+    //         //     title = title
+    //         //     }
+
+    //         let newEvent = {} as CalendarEvent;
+    //         newEvent.start = moment(start).toDate();
+
+    //         // TODO :: Fix 'All Day' Agenda for All day events
+    //         newEvent.end = moment(end).add(1, "seconds").toDate();
+    //         newEvent.title = title;
+
+    //         // Tracking previous events
+    //         setEvents([
+    //           ...events,
+    //           newEvent
+    //         ])
+    //     }
       }
 
-    // Returning calaendar state and events
+    // Returning calendar state and events
     return (
-      <>
-        <Calendar
-          selectable
-          localizer={localizer}
-          events={events}
-          defaultView='month'
-          views={allViews}
-          max={dates.add(dates.endOf(new Date(2015, 17, 1), 'day'), -1, 'hours')}
-          onSelectEvent={event => alert(event.title)}
-          onSelectSlot={handleSelect}
-          startAccessor='start'
-          endAccessor='end'
-          titleAccessor='title'
-        />
-      </>
+        // <div>
+        <>
+            <Calendar
+            selectable
+            localizer={localizer}
+            events={events}
+            defaultView='month'
+            views={allViews}
+            max={dates.add(dates.endOf(new Date(2015, 17, 1), 'day'), -1, 'hours')}
+            onSelectEvent={event => alert(event.title)}
+            onSelectSlot={handleSelect}
```

Comment:
We don't need to do this just yet, but this will need to be a handleSelect and handleCreate, in order for us to pass some sort of Event object into the modal, either a new event or an existing event, to view/edit/create. We should make this into a task for this iteration